region/maridia/inner-pink/Colosseum.json

```diff
@@ -1194,6 +1194,283 @@
+                "canMomentumConservingTurnaround"
+              ],
+              "note": ["This applies to Double Chamber."],
+              "devNote": [
+                "The jump could also be done with a shorter runway, with a tricky dash jump using run speed $4.0 or $4.1.",
+                "But it wouldn't apply to any more rooms in the game."
+              ]
+            },
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 23.8731,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "environment": "water",
+              "requires": [
+                "canInsaneJump",
+                "canGravityJump",
+                "canMomentumConservingTurnaround"
+              ],
+              "note": [
+                "This applies to Below Botwoon Energy Tank and Botwoon Energy Tank Room",
+                "gain run speed, and time a pause to unequip Gravity after Samus jumps;",
+                "hold down and back through the unpause to buffer a turnaround."
+              ],
+              "detailNote": [
+                "With a momentum conserving turnaround, this has a 3-frame window for the jump,",
+                "and between a 5-frame and 7-frame window for the pause, depending on the jump timing",
+                "(with later jumps corresponding to larger windows for the pause to hit).",
+                "A ceiling mockball is also possible but with greater difficulty."
+              ]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 45,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canInsaneJump",
+                {"or": [
+                  "canMomentumConservingMorph",
+                  "canMomentumConservingTurnaround"
+                ]}
+              ],
+              "note": ["This applies to Statues Hallway and Baby Kraid Room."],
+              "devNote": [
+                "With a momentum conserving morph, this has a 2-frame window for the jump,",
+                "and a 1-frame or 2-frame window for the morph depending on the jump timing",
+                "(with a last-frame jump giving the larger window for the morph).",
+                "With a turnaround, this requires a last-frame jump, with a 2-frame window for the turnaround."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 21.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "environment": "water",
+              "requires": [
+                "canTrickyDashJump",
+                "canInsaneJump",
+                "canGravityJump",
+                "canMomentumConservingTurnaround"
+              ],
+              "note": [
+                "This applies to Draygon's Room.",
```

Comment:
This took a lot of playing around with to see where the extra height comes from.  moving forward and back, pausing later jumping earlier timing the turnaround.  So its trickier to understand than others, but not particularly hard.

Reply:
I think it still fits.